ibis/tests/expr/test_value_exprs.py

```diff
@@ -626,20 +627,23 @@ def test_null_column_union():
     assert s.union(t.mutate(b=ibis.NA.cast('double'))).schema() == s.schema()
 
 
-def test_string_compare_numeric_array(table):
-    with pytest.raises(TypeError):
-        table.g == table.f  # noqa: B015
+# TODO(kszucs): rethink how to handle this, theoretically the user should be
+# able to compare a string column with a numeric literal which would evaluate
+# to False for all rows
+# def test_string_compare_numeric_array(table):
+#     with pytest.raises(ValidationError):
+#         table.g == table.f
```

Comment:
These should be uncommented before merging.

Reply:
Restored.